subworkflows/local/quality_control/main.nf

```diff
@@ -2,7 +2,7 @@
 // Perform quality control
 //
 
-include { NACHO_QC } from '../../../modules/local/nacho/qc'
+include { NACHO_QC } from '../../../modules/local/nacho/qc/main.nf'
```

Comment:
You could have left this unchanged, it gets read with or without the main.nf, not that it matters too much